### PR TITLE
Add Convex Logo

### PR DESCRIPTION
**Pull request by nipunn1313**
See
https://github.com/get-convex/convex-backend/blob/main/LICENSE.md

https://news.convex.dev/convex-goes-open-source/

### CODE REVIEW

**Comment by chadwhitacre** [comment]
Heyooooo! Nice! Welcome aboard! 😁 💃 

Heads up that you will likely get pushback from the community on calling an FSL-licensed project "Open Source." [We went through this last year with Codecov](https://blog.sentry.io/lets-talk-about-open-source/). We have been working since then to find the right term to use, and over on [howtoshare.software](https://howtoshare.software/) (still in draft—meeting with a designer tomorrow 😬 ) we are arriving at "Fair Source" as the preferred term. Obviously up to you how you want to play it, but I don't want you to be surprised if you get pushback that might overshadow what should be pure celebration. 🥳 😁

File: index.html [hunk 1]
```diff
@@ -83,6 +83,7 @@ <h2>Who?</h2>
             <li><a title="Answer Overflow" href="https://github.com/AnswerOverflow/AnswerOverflow" ><img src="./logos/answer-overflow.svg" alt="Answer Overflow logo" /></a></li>
             <li><a title="CodeCrafters" href="https://github.com/codecrafters-io/frontend" ><img src="./logos/codecrafters.svg" alt="CodeCrafters logo"/></a></li>
             <li><a title="Codecov" href="https://github.com/codecov/self-hosted" ><img src="./logos/codecov.svg" alt="Codecov logo" /></a></li>
+            <li><a title="Convex" href="https://convex.dev" ><img src="./logos/convex.svg" alt="Convex logo" /></a></li>
```
Comment (chadwhitacre) [inline index.html:86]: You up for using https://github.com/get-convex/convex-backend here?

Comment (nipunn1313) [reply index.html:86]: Yeah that's also reasonable!

Comment (nipunn1313) [reply index.html:86]:
```suggestion
            <li><a title="Convex" href="https://github.com/get-convex/convex-backend" ><img src="./logos/convex.svg" alt="Convex logo" /></a></li>
```

**Comment by nipunn1313** [comment]
updated the link!

**Comment by dcramer** [comment]
👋  @nipunn1313

**Comment by chadwhitacre** [comment]
Thanks @nipunn1313! Welcome aboard! 😁

**Pull request merged**